src/tools/fuzzing/heap-types.cpp

```diff
@@ -109,21 +108,18 @@ struct HeapTypeGenerator {
   Type generateTupleType() {
     std::vector<Type> types(2 + rand.upTo(MAX_TUPLE_SIZE - 1));
     for (auto& type : types) {
-      // Make sure tuples are defaultable. See comment in
-      // TranslateToFuzzReader::getTupleType.
```

Comment:
It might if we were going to create function bodies for these function types, but we don't necessarily need to do that. Also, @kripken has been updating passes to work with non-nullable locals, so that comment in fuzzing.cpp is possibly out of date.